src/core/Akka.MultiNodeTestRunner/Akka.MultiNodeTestRunner.csproj

```diff
@@ -40,6 +42,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\packages\xunit.abstractions.2.0.0\lib\net35\xunit.abstractions.dll</HintPath>
     </Reference>
+    <Reference Include="xunit.core">
```

Comment:
Added a reference to XUnit core... an assembly which the runner was depending on the tests for previously (DLL hell issue)